script/Deploy.s.sol

```diff
@@ -30,7 +30,9 @@ contract Deploy is ZoraDropsDeployBase {
 
         console2.log("Setup contracts ---");
 
-        vm.startBroadcast();
+        address deployer = vm.envAddress("DEPLOYER");
+
+        vm.startBroadcast(deployer);
```

Comment:
+1 here, nice fix. was thinking of doing this instead of passing in `--sender` explicitly.